ccloader/js/ccloader.js

```diff
@@ -266,6 +274,8 @@ export class ModLoader {
 
 			if (!enabled) {
 				result[depName] = `${depDesc} is disabled`;
+			} else if (isExtension && this.extensions.indexOf(depName) === -1) {
```

Comment:
This branch should be placed above the `enabled` check (so that extensions can't be "disabled").